install/kubernetes/helm/istio/values-sidecar-injector.yaml

```diff
@@ -0,0 +1,26 @@
+# This is used to generate istio.yaml
```

Comment:
Settings for sidecar should go under 'sidecar-injector' section in values.yaml, like all other features.

Reply:
This is for generating separate sidecar-injector.yaml which is using only the chart under sidecar-injector sub-directory.

Reply:
@yusuoh I'm not sure that will work, as the .tpl file is in the top level chart (as per consolidation requests in previous PRs).  I can understand the rationale - perhaps we should duplicate those tpls specifically for side-car for this use case.  Have you tested this PR as is? (I could be wrong about it not working is why I ask)